tests/framework/enums/rook_platform_types.go

```diff
@@ -1,63 +0,0 @@
-/*
-Copyright 2016 The Rook Authors. All rights reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-	http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package enums
-
-import (
-	"fmt"
-	"strings"
-)
-
-//RookPlatformType is enum if rook platforms
-type RookPlatformType int
-
-const (
-	//Kubernetes plafrom is default
-	Kubernetes RookPlatformType = iota + 1
-	//BareMetal plafrom
-	BareMetal
-	//StandAlone plafrom is default
-	StandAlone
```

Comment:
To fully remove standalone (in another PR), it would also be good to remove the unnecessary test interfaces. Navigating the code will be a lot easier that way.